lib/rubocop/cop/rspec/factory_bot/syntax_methods.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use the shorthands from `FactoryBot::Syntax::Methods` in your specs.
+        #
+        # The auto-correction is marked as unsafe because the cop
+        # cannot verify whether you already include
+        # `FactoryBot::Syntax::Methods` in your test suite.
+        #
+        # If you're using Rails, add the following configuration to
+        # `spec/support/factory_bot.rb` and be sure to require that file in
+        # `rails_helper.rb`:
+        #
+        # [source,ruby]
+        # ----
+        # RSpec.configure do |config|
+        #   config.include FactoryBot::Syntax::Methods
+        # end
+        # ----
+        #
+        # If you're not using Rails:
+        #
+        # [source,ruby]
+        # ----
+        # RSpec.configure do |config|
+        #   config.include FactoryBot::Syntax::Methods
+        #
+        #   config.before(:suite) do
+        #     FactoryBot.find_definitions
+        #   end
+        # end
+        # ----
+        #
+        # @example
+        #   # bad
+        #   FactoryBot.create(:bar)
+        #   FactoryBot.build(:bar)
+        #   FactoryBot.attributes_for(:bar)
+        #
+        #   # good
+        #   create(:bar)
+        #   build(:bar)
+        #   attributes_for(:bar)
+        #
+        class SyntaxMethods < Base
+          extend AutoCorrector
+          include InsideExampleGroup
+          include RangeHelp
+
+          MSG = 'Use `%<method>s` from `FactoryBot::Syntax::Methods`.'
+
+          RESTRICT_ON_SEND = %i[
+            attributes_for
+            attributes_for_list
+            attributes_for_pair
+            build
+            build_list
+            build_pair
+            build_stubbed
+            build_stubbed_list
+            build_stubbed_pair
+            create
+            create_list
+            create_pair
+            generate
+            generate_list
+            null
+            null_list
+            null_pair
+          ].to_set.freeze
+
+          # @!method factory_bot_method?(node)
+          def_node_matcher :factory_bot_method?, <<~PATTERN
+            (send (const {nil? cbase} :FactoryBot) $%RESTRICT_ON_SEND ...)
+          PATTERN
+
+          def on_send(node)
+            return unless inside_example_group?(node)
+
+            factory_bot_method?(node) do |method|
+              message = format(MSG, method: method)
+
+              add_offense(crime_scene(node), message: message) do |corrector|
+                corrector.remove(offense(node))
+              end
+            end
+          end
+
+          private
+
+          def crime_scene(node)
```

Comment:
I think I've first seen `evidence` used for the first time in RuboCop ecosystem, and now `crime_scene`. Those are perfect names to describe what those methods do.